app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_names = command.split()
+    if len(file_names) != 3 or file_names[0] != "cp":
+        return
+
+    if file_names[1] == file_names[2]:
+        return
+
+    try:
+        with (
+            open(file_names[1], "r") as file_in,
+            open(file_names[2], "w") as file_out
+        ):
+            file_out.write(file_in.read())
```

Comment:
Now it`s not a problem, but when you need to copy very big file it can be a problem, because when you start copy, it takes much memory, better copy the file by rows. 
For example:
`for line in file_in:
    file_out.write(line)`